core/templates/dev/head/pages/exploration-editor-page/services/user-email-preferences.service.spec.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the UserEmailPreferencesService.
+ */
+
+import { UpgradedServices } from 'services/UpgradedServices';
```

Comment:
Hi @Arnesh07, Please add a TODO here to remove this line once angular migration is complete.

Reply:
Hey! @nishantwrp Thanks for the review. I have done this, PTAL!